### DISCUSSION TITLE
How to use --at-apply

### DISCUSSION THREAD

**Discussion by wfion** [unocss/unocss#2332] [Q&A]
![20230308_142400_s2IM8wly3h.png](https://s2.232232.xyz/static/422/2023/03/08-64082ba9a1598.png)

it's not work in playground

**Comment by Monns** [comment]
add this package

https://github.com/unocss/unocss/tree/main/packages/transformer-directives

**Reply by wfion** [reply]
Thank you~ It's work!

**Comment by zyyv** [comment]
`transformer-directives` will be used together with the `vite plugin`, please make sure your develop environment includes `vite`.

Our playground generated css within UnoCSS original api, not in vite, so cann't use directives in playground.

**Reply by wfion** [reply]
Thank you~ It's work!

**Reply by mdxiaohu** [reply]
`transformer-directives` Can't webpack be supported .

**Reply by ChenWeihua123** [reply]
transformer-directives Can't webpack be supported . +1
